# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uisp/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- uisp-20040311/src/Global.h	2003-02-17 18:16:14.000000000 +0100
+++ uisp-20040311/src/Global.h	2005-01-05 19:22:36.000000000 +0100
@@ -25,6 +25,10 @@
 
 /* Global.h, Uros Platise (c) 1999 */
 
+#ifndef NULL
+#define NULL 0
+#endif
+
 #ifndef __GLOBAL
 #define __GLOBAL
 
